src/System.Net.Security/src/System/Net/SslStreamPal.Windows.cs

```diff
@@ -33,6 +33,7 @@ public static Exception GetException(SecurityStatusPal status)
         }
 
         internal const bool StartMutualAuthAsAnonymous = true;
+        internal const bool PresizeEncryptBuffer = true;
```

Comment:
@bartonjs If it's not simple to remove this const, please add a comment. Pasting a portion of the explanation from this PR's description should be enough to get context in the future.